.github/CODEOWNERS

```diff
@@ -84,4 +82,4 @@
 
 # DevOps
 *.yml @awentzel @janechu @nicholasrice @chrisdholt
-build/ @janechu @nicholasrice @chrisdholt
\ No newline at end of file
+build/ @janechu @nicholasrice @chrisdholt
```

Comment:
I should be added to this owners group and the Title would be more accurate as either "Build scripts".  There are things in here for cleanup, testings, publishing etc.

Reply:
Done.